### PR TITLE
Lesson9 Mihaela Vaida

### PR DESCRIPTION
**Pull request by MihaelaGit**

### CODE REVIEW

File: lesson_9/lesson_9_mihaela_vaida/tema_inheritance_mihaela_vaida.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+        return cls(first_name, last_name, age, studies, grade)
+
+    def description(self):
+        return f'The employee {self.first_name} {self.last_name[0]}. age: {self.age}'
+
+    def is_promoted(self):
+        return self.grade > 7 and self.age <= 60
+
+    def __repr__(self):
```
Comment (Liitle) [inline lesson_9/lesson_9_mihaela_vaida/tema_inheritance_mihaela_vaida.py:27]: Nice

File: lesson_9/lesson_9_mihaela_vaida/tema_inheritance_mihaela_vaida.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        return f'Employee: {self.first_name} {self.last_name} with id: {id(self)}'
+
+    def __str__(self):
+        return f'Employee: {self.first_name} {self.last_name[0]}.'
+
+
+class InstructedEmployee(Employee):
+    def __init__(self, first_name, last_name, age, studies, grade, courses):
+        super().__init__(first_name, last_name, age, studies, grade)
+       # self.first_name = first_name
```
Comment (Liitle) [inline lesson_9/lesson_9_mihaela_vaida/tema_inheritance_mihaela_vaida.py]: Usually commented code can be removed

**Review by Liitle — COMMENTED** [review]
Nice work, clean and to the point.

**Review by Liitle — APPROVED** [review]

File: lesson_9/lesson_9_mihaela_vaida/tema_inheritance_mihaela_vaida.py (hunk 2 above)
Comment (MihaelaGit) [reply lesson_9/lesson_9_mihaela_vaida/tema_inheritance_mihaela_vaida.py]: I`ll remove the commented lines

**Review by mariuspodean — APPROVED** [review]

**Pull request merged**